apps/imb-mpi/ringpingpong.pbs

```diff
@@ -2,24 +2,31 @@
 
 # setup Intel MPI environment for Infiniband
 source /etc/profile # so we can load modules
-module load mpi/impi_2018.4.274
+module load mpi/impi
 source $MPI_BIN/mpivars.sh
 
 hostlist=$(pwd)/hosts.$PBS_JOBID
 
 sort -u $PBS_NODEFILE > $hostlist
 
+export I_MPI_FABRICS="shm:dapl"
+export I_MPI_DYNAMIC_CONNECTION=0
```

Comment:
this is only needed for h16r (and due to the lack of support from the microsoft driver)

Reply:
got it